### PR TITLE
Fixed webgl error, fixed inventory issue and some UI changes

### PR DESCRIPTION
**Pull request by Krayt78**

### CODE REVIEW

File: Assets/AnkrDemo/3D Demo/Scripts/DemoScript_Scifi.cs [hunk 1]
```diff
@@ -204,7 +204,12 @@ private async void ApproveCharacterCall()
 			await _contractHandler.ApproveAllForCharacter(true);
 		}
 
-		private async void OnLoadButtonClickedCall()
+		private void OnLoadButtonClickedCall()
```
Comment (Vukiz) [inline Assets/AnkrDemo/3D Demo/Scripts/DemoScript_Scifi.cs]: Don't use async void! 
Use UniTask with .Forget!

**Review by Vukiz — CHANGES_REQUESTED** [review]

**Comment by Vukiz** [comment]
obj/Debug/Assembly-CSharp-firstpass.csproj.AssemblyReference.cache
add obj/ folder to gitignore

**Review by Vukiz — APPROVED** [review]

File: Assets/AnkrDemo/Scripts/DemoContractHandler.cs [hunk 2]
```diff
@@ -6,23 +7,27 @@
...
+using UnityEngine.Serialization;
 
 namespace Demo.Scripts
 {
 	public class DemoContractHandler : MonoBehaviour
 	{
 		private const string TransactionGasLimit = "1000000";
 
-		[SerializeField] private TMP_Text _text;
+		[FormerlySerializedAs("_text")] [SerializeField] private TMP_Text text;
```
Comment (Vukiz) [inline Assets/AnkrDemo/Scripts/DemoContractHandler.cs]: why?

File: Assets/AnkrDemo/Scripts/WebGLConnectionController.cs [hunk 3]
```diff
@@ -47,6 +48,7 @@ private void OnDisable()
 			_webGlLoginManager.NetworkChosen -= OnNetworkChosen;
 			_webGlLoginManager.WalletChosen -= OnWalletChosen;
 		}
-	}
 #endif
+	}
+
```
Comment (Vukiz) [inline Assets/AnkrDemo/Scripts/WebGLConnectionController.cs]: extra space

**Review by Vukiz — CHANGES_REQUESTED** [review]

**Comment by Vukiz** [comment]
Remove obj/ from git index

File: Assets/AnkrDemo/Scripts/DemoContractHandler.cs (hunk 2 above)
Comment (Krayt78) [reply Assets/AnkrDemo/Scripts/DemoContractHandler.cs]: uh might be my IDE that did that whenever i tried to rename another variable

**Review by Vukiz — APPROVED** [review]

**Pull request merged**